tests/integration_tests/test_applications.py

```diff
@@ -259,7 +259,7 @@
     "generate_corsika_histograms::output":
         [["--IACT_file", "./tests/resources/"
                          "tel_output_10GeV-2-gamma-20deg-CTAO-South.corsikaio",
-          "--png", "--ecsv", "--output_directory", "."]],
+          "--png", "--ecsv", "--output_path", "."]],
```

Comment:
Pointing to `.` will have to change later to the temp directory but there's an open issue about it and I want to take a look in general so for now leave it like that and we fix it later.